pages/common/gcloud.md

```diff
@@ -0,0 +1,32 @@
+# gcloud
+
+> The official CLI tool for Google Cloud Platform. You can use this tool to perform many common platform tasks either from the command-line or in scripts and other automations.
```

Comment:
Due to the current length, perhaps split this into 2 lines. Something like:

> The official CLI tool for Google Cloud Platform.
> This tool allows you to perform common platform tasks.

Reply:
SGTM